bfh/fields.py

```diff
@@ -48,7 +48,7 @@ def validate(self, value):
                 return False
 
     """
-    def __init__(self, required=True):
+    def __init__(self, default=None, required=True):
```

Comment:
when inserting a kwarg in the signature i'd prefer putting it at the end; in python you can pass values to kwargs positionally, so you're changing the signature in a breaking manner here if anyone is doing `foo = SomeField(False)`

```python
In [1]: def foo(one, two=2):
    print one, two
   ...:

In [2]: foo(1)
1 2

In [3]: foo(1, 3)
1 3
```